### PR TITLE
Upgrade Express Dependency

### PR DESCRIPTION
**Pull request by keriwarr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/ADEHJKNTV/Yung.js.git"
   },
   "dependencies": {
-    "express": "^4.0.0"
+    "express": "^3.0.0"
```
Comment (dhop) [inline package.json:12]:
```suggestion
    "express": "^3.0.1"
```

**Review by grcooper — APPROVED** [review]

**Review by dhop — CHANGES_REQUESTED** [review]
You call that an upgrade?

File: package.json (hunk 1 above)
Comment (grcooper) [reply package.json:12]: How dare u. We must stay pure with `3.0.0`

**Comment by keriwarr** [comment]
`question` label is a nicer color, thanks!

**Pull request closed without merging**